### PR TITLE
CS-2241: Added extra library headers

### PR DESCRIPTION
**Pull request by kc-sabal**

### CODE REVIEW

File: src/main/java/co/dapi/DapiRequest.java [hunk 1]
```diff
@@ -15,6 +15,9 @@ public class DapiRequest {
     public final static String Dapi_URL = "https://api.dapi.com";
     public final static String DD_URL = "https://dd.dapi.com";
     public final static String SECURE_DD_URL = "https://dd.secure.dapi.com";
+    public final static String LIBRARY_VERSION = "1.6.0";
+    public final static String LIBRARY = "dapi-java";
```
Comment (asmsh) [inline src/main/java/co/dapi/DapiRequest.java]: Make them private.

File: src/main/java/co/dapi/DapiRequest.java [hunk 2]
```diff
@@ -58,6 +61,13 @@ private static Response doRequest(String bodyJson, String url, HashMap<String, S
             headersBuilder.set(header.getKey(), header.getValue());
         }
         headersBuilder.set("content-type", "application/json");
+        Package objPackage = co.dapi.DapiApp.class.getPackage();
+
+        String version = objPackage.getImplementationVersion() != null ? objPackage.getImplementationVersion() : LIBRARY_VERSION;
+        String artifactId = objPackage.getImplementationTitle() != null ? objPackage.getImplementationTitle() : LIBRARY;
+        headersBuilder.set("X-Dapi-Library-Version", version);
+        headersBuilder.set("X-Dapi-Library", artifactId);
```
Comment (asmsh) [inline src/main/java/co/dapi/DapiRequest.java]: Use the defined static values directly.

File: src/main/java/co/dapi/DapiRequest.java [hunk 3]
```diff
@@ -58,6 +61,13 @@ private static Response doRequest(String bodyJson, String url, HashMap<String, S
...
+
```
Comment (asmsh) [inline src/main/java/co/dapi/DapiRequest.java]: Extra new line.
```suggestion
```

**Review by asmsh — CHANGES_REQUESTED** [review]

File: src/main/java/co/dapi/DapiRequest.java (hunk 1 above)
Comment (kc-sabal) [reply src/main/java/co/dapi/DapiRequest.java]: https://github.com/dapi-co/dapi-java/pull/21/commits/2560ac886823fa1ef0d1b6793ea3b64af76fe809

File: src/main/java/co/dapi/DapiRequest.java (hunk 2 above)
Comment (kc-sabal) [reply src/main/java/co/dapi/DapiRequest.java]: https://github.com/dapi-co/dapi-java/pull/21/commits/574895364b30eec5150ab5f69062a55c1f6f1e4a

File: src/main/java/co/dapi/DapiRequest.java (hunk 3 above)
Comment (kc-sabal) [reply src/main/java/co/dapi/DapiRequest.java]: Removed here https://github.com/dapi-co/dapi-java/pull/21/commits/574895364b30eec5150ab5f69062a55c1f6f1e4a

**Review by asmsh — APPROVED** [review]

**Pull request merged**